snuba/migrations/snuba_migrations/replays/0005_add_urls_user_agent_replay_start_timestamp.py

```diff
@@ -0,0 +1,84 @@
+from typing import List, Sequence, Tuple
+
+from snuba.clickhouse.columns import Array, Column, DateTime, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+new_columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (Column("urls", Array(String())), "url"),
```

Comment:
I don't think this really matters, but I wouldn't make the "after" a column that is being removed.